packages/jsii-dotnet-analyzers/src/Amazon.JSII.Analyzers/Amazon.JSII.Analyzers.csproj

```diff
@@ -10,9 +10,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="Amazon.JSII.Runtime" Version="$(JsiiVersion)" />
-      <PackageReference Include="Microsoft.CodeAnalysis" Version="3.2.0" />
-      <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.2.0" />
+      <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.2.0" PrivateAssets="all" />
```

Comment:
Not sure why this needs to be PrivateAssets="all".

Reply:
This is part of the (Microsoft-authored) tutorial I have found... So I fixed it up to match... I reckon the reason is that the compiler actually comes with it's own and it's better to not pollute, but I could be wrong.